### PR TITLE
Add actions and reducers for books and categories

### PR DESCRIPTION
**Pull request by erickhansx**
## Checklist before requesting a review
- [ ]Define action types for adding and removing a book.
- [x] Set the initial state to be an empty array of books.
- [x] Export Action Creators for your actions.
- [x] Write your reducer and export it as default.
- [x] Importing the necessary methods from Redux Toolkit.
- [x] Combine both reducers into a root reducer by using configureStore function.

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/books';
+
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
+  },
+});
+
+export default store;
```
Comment (GzimAsani) [inline src/redux/configureStore.js]: - [x] Please import the categories and add them on the configurestore just as you did with books

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @erickhansx              :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job implementing the reducers :+1: 

 ## Required Changes 


- [ ] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/configureStore.js (hunk 1 above)
Comment (erickhansx) [reply src/redux/configureStore.js]: Thank you so much for your observation, I forgot to do that!

**Review by m05e5 — APPROVED** [review]
Hi @erickhansx , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
goodjob

## Highlights
✔️ your linters are all ok
✔️ Descriptive PR 

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**